### PR TITLE
Substitute configure command token in the walkthrough

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/reboot/issues/257.

## Description
This PR changes the `[CONFIGURE_COMMAND]` placeholder in the walkthrough to a personalized token install command.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ gem 'bootsnap', '>= 1.1.0', require: false
 
 group :development, :test do
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'capybara', '~> 2.13.0'
+  gem 'capybara'
```
Comment (iHiD) [inline Gemfile]: Is it worth specifying that there's a requirement for `~3.3` here?

File: test/test_helper.rb [hunk 2]
```diff
@@ -16,6 +16,17 @@ class ActionView::TestCase
...
+
+  def stub_repo_cache!(&block)
+    git_repo_cache = "#{Rails.root}/test/tmp/git_repo_cache"
+    Git::RepoBase.
+      stubs(:repo_base_dir).
+      returns(git_repo_cache)
+
+    yield
+
+    FileUtils.rm_rf(git_repo_cache)
```
Comment (iHiD) [inline test/test_helper.rb]: Is it worth wrapping this in an `ensure`?

File: test/test_helper.rb [hunk 3]
```diff
@@ -39,4 +50,15 @@ def repo_mock
...
+
+  def using_repo_cache!(&block)
+    git_repo_cache = "#{Rails.root}/test/tmp/git_repo_cache"
+    Git::RepoBase.
+      stubs(:repo_base_dir).
+      returns(git_repo_cache)
+
+    yield
+
+    FileUtils.rm_rf(git_repo_cache)
```
Comment (iHiD) [inline test/test_helper.rb]: Same here about the `ensure`

File: Gemfile (hunk 1 above)
Comment (kntsoriano) [reply Gemfile]: Good catch. Let's use `~> 3.0` as specified [here](https://github.com/rails/rails/pull/30959#discussion_r181931451).

File: test/test_helper.rb (hunk 2 above)
Comment (kntsoriano) [reply test/test_helper.rb]: Yes! Thank you :)

**Pull request merged**